examples/express.js

```diff
@@ -0,0 +1,116 @@
+const permissionary = require('..');
+const app = require('express')();
+const error = require('http-errors');
+const serialize = require('serialize-http-error');
+const json = require('body-parser').json();
+const {readFile} = require('fs');
+const grants = require('./grants');
+
+// In this example, the grants are stored statically. You could create a
+// dynamically configurable permission system by loading the grants from a
+// database at request time.
+const createVerifier = permissionary(grants);
+
+// This is our authentication middleware.
+app.use((req, res, next) => {
+
+  // This is where we would query our database for the currently logged-in
+  // user, and assign them their permissions. We could use Passport or
+  // anything else to determine which user this should be. For the purpose of
+  // this example, authentication is done by setting a user ID in a header.
+  const userId = Number(req.headers['authenticated-user']);
+
+  readFile('./users.json', 'utf8', function(err, res){
```

Comment:
Why `readFile('./users.json', ...)` rather than `require('./users.json')`?

Reply:
For the purpose of showing something async to be more like how it would be when loading from a database.

Reply:
Ah, yes.